### ISSUE TITLE
Add tradeskill cooldowns daily/weekly

### ISSUE THREAD

**Issue by chancedj** [chancedj/LockedOut#44]

**Comment by chancedj** [comment]
do you have any of the questId's for these quests?

**Comment by Daeveren** [comment]
Which of the quests? The tradeskill cooldowns?

**Comment by chancedj** [comment]
yes.  i can probably find a few.  but if you have a couple you want tracked just let me know and i'll see about adding them. the name or the quest/spelll id

**Comment by Daeveren** [comment]
Well, I've been trying to find them on wowhead, but to no avail. It seems that wowhead does not have a filter for "has cooldown". The full list of spells that create a tradeskill item as an effect is 232 items, but there's no way (that I found) to further filter by which has a daily CD -> http://www.wowhead.com/profession-spells?filter=109;157;0

Ingame I see a list inside TSM, it tracks the profession CDs. These must be all of them.

![2017-10-18 16_09_06-world of warcraft](https://user-images.githubusercontent.com/6321337/31720249-b1f0d09e-b41e-11e7-8282-370a9f97626d.png)

![2017-10-18 16_09_21-world of warcraft](https://user-images.githubusercontent.com/6321337/31720263-bc86dc60-b41e-11e7-9e99-f8945ae80ece.png)

**Comment by chancedj** [comment]
thanks.  though if there is an addon that does this a lot better, should i still aim to add it into mine? i'll check to see if they expose their data for other addons and see about adding the data into mine if that's the case.  outside of that, i'm wondering if i should be tracking ALL of these, or ones that have a less visual indicator elsewhere (e.g.: the jc quest in dalaran for the daily jewel). thoughts?

**Comment by Daeveren** [comment]
Well, these above are profession cooldowns and it would be great info to have in LockedOut --- but I think it would be best to show them in the basic form (which I believe SavedInstances was doing too). For example, alchemy puts like 20 transmutes on the *same* CD when you use one, so LO would only show the specific transmute you used (as a tracked profession CD in the tooltip) or just display something generic like "Alchemy - Transmutes". What I can tell is that many/most (if not all) of the goldmakers are doing the WoD daily profession CDs everyday (some still do the MoP ones, but not many).
Maybe we could work something out as in grouping all the spells in the list above in generic (but very useful, actually) categories such as Tailoring - Dreamcloth, Jewelcrafting - Research, Alchemy - Transmutes.
Also some of the ones in the list above are garrison 'quests' so I'm not even sure LO should even track them.
I could come with the spell ID for the profession CDs that I use if it's useful, but then, other players might use other CDs that I don't.

**Comment by chancedj** [comment]
going to try and get this one added into 1.7.0

**Comment by chancedj** [comment]
moving off 1.7 as it became a bigger release than i wanted.  but still on the radar for the next couple releases.
